system/Test/Mock/MockInputOutput.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Test\Mock;
+
+use CodeIgniter\CLI\InputOutput;
+use CodeIgniter\Test\Filters\CITestStreamFilter;
+use CodeIgniter\Test\PhpStreamWrapper;
+use InvalidArgumentException;
+use LogicException;
+
+class MockInputOutput extends InputOutput
+{
+    /**
+     * String to be entered by the user.
+     *
+     * @var array<int, string>
+     * @phpstan-var list<string>
```

Comment:
I think we just decided to consolidate these, right?
```suggestion
     * @var list<string>
```